<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="gkillare-win87" failures="1" tests="24" name="testcases.TC_CAN_VoidOfSale" time="1252.612" errors="0" timestamp="2024-06-19T16:36:32 IST" skipped="0">
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard06" time="60.095"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard13" time="61.070"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard16" time="57.128"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard11" time="41.158"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard01" time="34.630"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard07" time="21.002"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard12" time="42.284"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard05" time="41.246"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard03" time="51.666"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard01" time="56.659"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard11" time="66.921"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard15" time="67.027"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard02" time="47.387"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard12" time="62.577"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard05" time="57.487"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard15" time="39.641"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard10" time="42.219"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard06" time="48.679"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard13" time="43.993"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard04" time="64.698"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard09" time="81.003"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard10" time="65.963"/>
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard14" time="39.017">
    <failure message="We are not getting ResponseText as APPROVAL or VALIDATION [null,,374245XXXXX1006,F,1,,AXC,null,0.00,050-DECLINED,20006,291241712413030306,000000000000000000,000000,374245XXXXX1006,AID Not Receive ,AID Not Receive ] expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: We are not getting ResponseText as APPROVAL or VALIDATION [null,,374245XXXXX1006,F,1,,AXC,null,0.00,050-DECLINED,20006,291241712413030306,000000000000000000,000000,374245XXXXX1006,AID Not Receive ,AID Not Receive ] expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at base.BaseClass.performSaleTransaction(BaseClass.java:216)
at testcases.TC_CAN_VoidOfSale.TestCard14(TC_CAN_VoidOfSale.java:309)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TestCard14 -->
  <testcase classname="testcases.TC_CAN_VoidOfSale" name="TestCard08" time="59.062"/>
</testsuite> <!-- testcases.TC_CAN_VoidOfSale -->
